websites/docs-smoke-test/src/data/navigation.yaml

```diff
@@ -19,6 +19,8 @@
       path: /components/images
     - title: Code Examples
       path: /components/code-examples
+    - title: Multi-language code examples
```

Comment:
Second piece of feedback: Can we merge the code example pages into one  or two again?  There are three now which is confusing, especially because two have the same name. 

I suggest to have one page for the markdown based ones and one for all the variations from external files.